src/main/java/org/datadog/jmxfetch/Status.java

```diff
@@ -122,11 +120,11 @@ private String generateYaml() {
         return yaml.dump(status);
     }
 
-    private String generateJson() throws JsonProcessingException {
+    private String generateJson() throws IOException {
```

Comment:
note: this looks OK to me, but I'm not sure this is well covered in our tests.

Reply:
OK I will add some testing before merging this

Reply:
Actually I can see why there isn't much testing for this now - I don't really have time to refactor this for testability right now. Is there another way I can find if this still works the way it's supposed to?